test-e2e/utils.js

```diff
@@ -165,15 +165,18 @@ export const waitForPeers = (managers, { waitForDeviceInfo = false } = {}) =>
  * @param {import('../src/generated/rpc.js').DeviceInfo['deviceType']} [deviceType]
  * @returns {Promise<import('type-fest').ReadonlyTuple<MapeoManager, T>>}
  */
-export async function createManagers(count, t, deviceType) {
+export async function createManagers(
+  count,
+  t,
+  deviceType = 'device_type_unspecified'
+) {
   // @ts-ignore
   return Promise.all(
     Array(count)
       .fill(null)
       .map(async (_, i) => {
         const name = 'device' + i + (deviceType ? `-${deviceType}` : '')
         const manager = createManager(name, t)
-        // @ts-ignore TODO: remove after updating @mapeo/schema
```

Comment:
This test-only change was the only meaningful change that wasn't a find-replace.